IPython/terminal/ptutils.py

```diff
@@ -43,6 +43,11 @@ def _elide(string, *, min_elide=30):
     return '{}.{}\N{HORIZONTAL ELLIPSIS}{}.{}'.format(parts[0], parts[1][0], parts[-2][-1], parts[-1])
 
 
+def _adjust_completion_text_based_on_context(text, body, offset):
```

Comment:
Could the naming be a bit clearer about what this function does? 'adjust based on context' is rather generic. Or maybe add a comment explaining the idea.